shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -445,4 +517,12 @@ private boolean isDebuggable() {
   public void onFirstFrameRendered() {
     hideCoverView();
   }
+
+  /**
+   * The mode of the background of a {@code FlutterActivity}, either opaque or transparent.
+   */
+  public enum BackgroundMode {
+    opaque,
```

Comment:
Nit: It's worth also talking about how these are going to be used here.

https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required